src/main/java/org/spongepowered/common/mixin/plugin/MultiWorldCommandPlugin.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.mixin.plugin;
+
+import org.apache.commons.lang3.StringUtils;
+import org.spongepowered.asm.lib.tree.ClassNode;
+import org.spongepowered.asm.mixin.extensibility.IMixinConfigPlugin;
+import org.spongepowered.asm.mixin.extensibility.IMixinInfo;
+import org.spongepowered.common.SpongeImpl;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class MultiWorldCommandPlugin implements IMixinConfigPlugin {
+
+    private static final String MIXIN_PREFIX = "MixinCommand";
+
+    private boolean configState;
+
+    @Override
+    public void onLoad(String mixinPackage) {
+        this.configState = mixinPackage.endsWith(".multiworld");
+    }
+
+    @Override
+    public String getRefMapperConfig() {
+        return null;
+    }
+
+    @Override
+    public boolean shouldApplyMixin(String targetClassName, String mixinClassName) {
+        final String name = StringUtils.substringAfter(mixinClassName, MIXIN_PREFIX).toLowerCase();
+
+        Map<String, Boolean> patches = SpongeImpl.getGlobalConfig().getConfig().getCommands().getMultiWorldCommandPatches();
+        if (!patches.containsKey(name)) {
+            patches.put(name, false);
```

Comment:
Regardless of the decision, should a message be output to the command sender alerting them to which option is active, and how to swap between them?